contracts/WildcardSteward_v3.sol

```diff
@@ -415,67 +416,28 @@ contract WildcardSteward_v3 is Initializable {
         );
     }
 
-    // TODO: this function needs to be deprecated - only used in the tests
-    function patronageOwed(uint256 tokenId)
-        public
-        view
-        returns (uint256 patronageDue)
-    {
-
-            uint256 tokenTimeLastCollectedPatron
-         = timeLastCollectedPatron[assetToken.ownerOf(tokenId)];
-
-        if (tokenTimeLastCollectedPatron == 0) return 0;
-
-        uint256 owed = price[tokenId]
-            .mul(now.sub(tokenTimeLastCollectedPatron))
-            .mul(patronageNumerator[tokenId])
-            .div(31536000000000000000);
-
-        return owed;
-    }
-
-    // TODO: this function needs to be deprecated - only used in the tests
-    function patronageOwedWithTimestamp(uint256 tokenId)
-        public
-        view
-        returns (uint256 patronageDue, uint256 timestamp)
-    {
-        return (patronageOwed(tokenId), now);
-    }
-
     function patronageOwedPatron(address tokenPatron)
         public
         view
         returns (uint256 patronageDue)
     {
-        if (timeLastCollectedPatron[tokenPatron] == 0) return 0;
-
+        // if (timeLastCollectedPatron[tokenPatron] == 0) return 0;
```

Comment:
This check should either exist in both patronageOwedPatron and patronageDueBenefactor or neither, it only existing in one doesn't make sense imo since these functions are essentially the same.

The tests pass with or without it. I don't think we need an extra check as it will return zero in any case if timeLastCollectedPatron[tokenPatron] == 0

Reply:
> The tests pass with or without it. I don't think we need an extra check as it will return zero in any case if timeLastCollectedPatron[tokenPatron] == 0

Yeah, makes sense, we must ensure that `totalPatronOwnedTokenCost[tokenPatron]` has to be zero if `timeLastCollectedPatron[tokenPatron]` is zero